rero_mef/theme/views.py

```
@@ -29,8 +77,549 @@
+    """Load property titles from local common JSON schema files."""
+    schema_name = _SCHEMA_NAME_MAP.get(entity_name)
+    if not schema_name:
+        return {}
+
+    schema_path = (
+        Path(__file__).resolve().parents[1] / "jsonschemas" / "common" / schema_name
+    )
+    try:
+        with schema_path.open("r", encoding="utf-8") as fp:
+            schema = json.load(fp)
+    except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)